src/components/Kv/KvLightbox.vue

```diff
@@ -126,6 +126,7 @@ body.scroll-locked {
 	bottom: 0;
 	padding: 4.5rem 0;
 	overflow-y: scroll;
+	z-index: 1500;
```

Comment:
Applying a z-index on this element fixed the issue. Other edits were just to "group" z-indexes for this component.